app/user/contact_request.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass, field, fields
+from flask import escape
+
+from typing import List
+
+from notifications_utils.recipients import validate_email_address
+
+__all__ = [
+    'ContactRequest',
+]
+
+
+@dataclass
+class ContactRequest:
+    email_address: str = field()
+    tags: List[str] = field(default_factory=lambda: list())
+    name: str = field(default='')
+    message: str = field(default='')
+    user_profile: str = field(default='')
+    department_org_name: str = field(default='')
+    program_service_name: str = field(default='')
+    intended_recipients: str = field(default='')
+    main_use_case: str = field(default='')
+    main_use_case_details: str = field(default='')
+    friendly_support_type: str = field(default='')
+    language: str = field(default='en')
```

Comment:
Can check if it's `en` or `fr`. Maybe `None` as well?

Reply:
I think we can safely default it to EN, I don't see the advantage of None over the existing default just because it is a certain language whether FR or EN. There will be more changes around this area to come.